src/state_manager/chain_rand.rs

```diff
@@ -221,11 +189,17 @@ pub fn draw_randomness(
 ) -> anyhow::Result<[u8; 32]> {
     let mut state = Params::new().hash_length(32).to_state();
     state.write_i64::<BigEndian>(pers)?;
-    let vrf_digest = blake2b_256(rbase);
+    let vrf_digest = digest(rbase);
     state.write_all(&vrf_digest)?;
     state.write_i64::<BigEndian>(round)?;
     state.write_all(entropy)?;
     let mut ret = [0u8; 32];
     ret.clone_from_slice(state.finalize().as_bytes());
     Ok(ret)
 }
+
+/// Computes a 256-bit digest.
+/// See <https://github.com/filecoin-project/ref-fvm/blob/master/fvm/CHANGELOG.md#360-2023-08-18>
+pub fn digest(rbase: &[u8]) -> [u8; 32] {
```

Comment:
what is `rbase`?

Reply:
I guess it mean `base` array for `r`andomness